etc/ci/performance/README.md

```diff
@@ -0,0 +1,66 @@
+Servo Page Load Time Test
+==============
+
+# Prerequisites
+
+* Python3
+
+# Basic Usage
+
+## Prepare the test runner
+
+* Clone this repo
+* Download [tp5n.zip](http://people.mozilla.org/~jmaher/taloszips/zips/tp5n.zip), extract it to `page_load_test/`
```

Comment:
Actually, after looking at `prepare_manifest.sh`, we can use `page_load_test/tp5n/`, which will cut down the noise in the file and still allow for multiple test suites.